examples/launch/launch-sweeps/custom-scheduler/README.md

```diff
@@ -0,0 +1,42 @@
+## Use a scheduler job for sweeps
+
+Sweeps on Launch are far more customizable than standard sweeps. The sweep scheduling mechanism can be entirely replaced with a job! You can use pre-made schedulers or implement your own custom scheduler job. Examples of what is possible with custom sweep scheduler jobs are available in the [wandb/launch-jobs](https://github.com/wandb/launch-jobs) repo under `jobs/sweep_schedulers`. This guide shows how to create a custom sweep scheduler job. 
```

Comment:
We should link to launch and jobs here so users know where to look for info about jobs and launch

Reply:
Also, could probably just directly link to `https://github.com/wandb/launch-jobs/jobs/sweep_schedulers`